Rei app

LamrabateImran opened the pull request.

LamrabateImran committed:
by train

LamrabateImran committed:
modified

LamrabateImran committed:
Update rei.py

LamrabateImran committed:
reviews function

LamrabateImran committed:
models

LamrabateImran committed:
Update proxy.py

LamrabateImran committed:
Create tasks.py

LamrabateImran committed:
modified

LamrabateImran committed:
Rei.com

hamidnahli commented on items/utils.py:
```diff
@@ -57,16 +54,10 @@ def get_ld_json(response: requests.Response):
-def get_response(product_id,offset=0):
-    url = f'https://api.bazaarvoice.com/data/batch.json?passkey=thvpbov9ywkkl4nkhbeq0wm1i&apiversion=5.5&displaycode=15372-en_us&resource.q0=reviews&filter.q0=isratingsonly%3Aeq%3Afalse&filter.q0=productid%3Aeq%3A{product_id}&filter.q0=contentlocale%3Aeq%3Aen*%2Cen_US&sort.q0=submissiontime%3Adesc&stats.q0=reviews&filteredstats.q0=reviews&include.q0=authors%2Cproducts%2Ccomments&filter_reviews.q0 ...
+def parse_bazaarvoice_reviews(product_id,offset=0):
+    url = f'https://api.bazaarvoice.com/data/batch.json?passkey=thvpbov9ywkkl4nkhbeq0wm1i&apiversion=5.5&displaycode=15372-en_us&resource.q0=reviews&filter.q0=isratingsonly%3Aeq%3Afalse&filter.q0=productid%3Aeq%3A{product_id}&filter.q0=contentlocale%3Aeq%3Aen*%2Cen_US&sort.q0=submissiontime%3Adesc&stats.q0=reviews&filteredstats.q0=reviews&include.q0=authors%2Cproducts%2Ccomments&filter_reviews.q0 ...
```
move the `passkey` to `.env`

LamrabateImran replied:
Done !

hamidnahli commented on spiders/rei.py:
```diff
@@ -499,6 +489,6 @@ def get_product_review(self) -> List:
                     }
-                reviews.append(review)
-        return reviews
+                product_reviews.append(review)      
```
move this loop to `parse_bazaarvoice_reviews`. This function should return all reviews in a list

LamrabateImran replied:
Done !

LamrabateImran committed:
add load env

LamrabateImran committed:
Update utils.py

LamrabateImran committed:
bazzarvoice's function

need more modification !

LamrabateImran committed:
Update rei.py

LamrabateImran committed:
cleaning...

LamrabateImran committed:
check test.py

check out test.py instead of models.py
m gonna change them when its ready to run efficiently.

LamrabateImran committed:
cleaning...

- load credentials from env
- test from run.py

LamrabateImran committed:
Update run.py

LamrabateImran committed:
cleaning...

trying to fix proxy.py and models.py
